code/modules/mining/ores_coins.dm

```diff
@@ -17,6 +17,27 @@
 			user << "<span class='info'>Not enough fuel to smelt [src].</span>"
 	..()
 
+/obj/item/weapon/ore/Crossed(AM as mob|obj)

```

Comment:
`/obj/item/weapon/ore/Crossed(atom/movable/AM)`

please, ```as X|Y``` statements do nothing in procs.